x/supplier/genesis.go

```diff
@@ -13,6 +13,10 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	for _, supplier := range genState.SupplierList {
 		k.SetSupplier(ctx, supplier)
 	}
+	// Set all the claim
```

Comment:
Does it make sense to track the claim list removal here as well? (https://github.com/pokt-network/poktroll/pull/149#discussion_r1386286626)

Reply:
Adding this TODO: `// TODO_TECHDEBT(@Olshansk): Remove existing claims from genesis.`